.config/yamllint/config.yml

```diff
@@ -20,4 +20,4 @@ rules:
   hyphens:
     level: error
   line-length:
-    max: 160
+    max: 175
```

Comment:
Not really fond of making a global change here for the line length. 
Ansible defaults to `160` and I think we should respect that unless absolutely necessary, then the line length limit can be overwritten on a line-by line cases with `# noqa yaml[line-length]`